### PR TITLE
DDF-3544 Renamed QUALIFIER constant in ddf.catalog.content.data.ContentItem to fix code smell

### PR DESCRIPTION
**Pull request by emmberk**
#### What does this PR do?
This PR renames field `ddf.catalog.content.data.ContentItem#QUALIFIER` to prevent any misunderstanding/clash with field `ddf.catalog.content.data.impl.ContentItemImpl#qualifer`.
#### Who is reviewing it? 
@AzGoalie @peterhuffer 
#### Select relevant component teams: 
@codice/core-apis 
#### Choose 2 committers to review/merge the PR. 
@clockard 
@coyotesqrl 
#### How should this be tested?
Full build.
#### What are the relevant tickets?
[DDF-3544](https://codice.atlassian.net/browse/DDF-3544)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/core/catalog-core-api/src/main/java/ddf/catalog/content/data/ContentItem.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
   String DEFAULT_FILE_NAME = "content_store_file.bin";
 
-  String QUALIFIER = "qualifier";
+  String QUALIFIER_KEYWORD = "qualifier";
```
Comment (clockard) [inline catalog/core/catalog-core-api/src/main/java/ddf/catalog/content/data/ContentItem.java]: Missed this the first time through but this is part of the catalog api so we unfortunately can't change this constant name. DDF 3.0 maybe?

Comment (emmberk) [reply catalog/core/catalog-core-api/src/main/java/ddf/catalog/content/data/ContentItem.java]: I don't this this class is whitelisted: http://artifacts.codice.org/service/local/repositories/releases/content/ddf/docs/2.11.4/docs-2.11.4-Documentation.html#_catalog_whitelist.

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
```Yarn Error```

**Comment by emmberk** [comment]
build now

**Pull request merged**